app/exawind/exawind.cpp

```diff
@@ -3,6 +3,7 @@
 #include "OversetSimulation.h"
 #include "MPIUtilities.h"
 #include "mpi.h"
+#include "yaml-editor.hpp"
```

Comment:
this is minor but... we don't really use the `hpp` extension in the exawind projects. Just noting in case someone cares

Reply:
Good call. I didn't think about that.